labs/coding-101-rest-basics-ga/6.md

```diff
@@ -18,7 +18,7 @@ As specified earlier, you can see that the base URL for DNA Center is:
 https://{DNA-Center-Server}/api/v1
 ```
 
-The following call retrieves all of hosts on the specified server
+The following call retrieves all of hosts on the specified server:
 
 ```
 GET https://{DNA-Center-Server}/api/v1/host
```

Comment:
Since /host is a deprecated call in DNA Center, we're gonna have to figure out the "right" way to retrieve all hosts on the specified server. I think it's "retrieve network device information: GET /network-device, GET /network-device/{networkDeviceId}/config" -- I can help out here. Flagging so we don't forget.

Reply:
Thanks! I noticed that this was a deprecated call in the documentation and was going to ask you about that.

Reply:
I had a chance to talk to Hank, and he thinks it's fine to use /hosts in examples for now until we get guidance for the exact replacement, and it is still available in the latest version on the Sandbox. So the /hosts can stay here.